docs/source/index.rst

```diff
@@ -9,15 +13,27 @@ On a single machine dask increases the scale of comfortable data from
 *fits-in-memory* to *fits-on-disk* by intelligently streaming data from disk
 and by leveraging all the cores of a modern CPU.
 
-Users interact with dask either by making graphs directly or through the *dask
-collections* which provide larger-than-memory counterparts to existing popular
-libraries:
+Dask use cases
+==============
+
+The types of users who are likely to find dask most useful are as follows:
+
+* **Library developers:** Projects like `xray`_ (climate science) and `scikit-image`_ use dask to accelerate interactive analysis of large datasets in their domains.
+* **Dask-array users:** Many users leverage the dask.array abstraction within dask.
+* **Algorithm developers:** Academics design complex algorithms and need a way to implement them; dask provides a testbed for them in applications such as linear algebra, optimization, and machine learning. They want to work directly with dask graphs or dask.imperative, eschewing the higher level collections.
+* **Users with large datasets needing pandas-like processing:** Dask.dataframe is useful to process data files that are larger than can fit in available memory. NOTE: dask covers part, but not all, of the `pandas`_ API.
+* **Multiprocessing users:** Some people are using dask.bag to process large dumps of log files.  
```

Comment:
This text doesn't seem sufficiently concise or relevant to deserve such a prominent place in the documentation.